images/microOS/config.xml

```diff
@@ -560,31 +117,21 @@
         <type
             image="iso"
             primary="true"
-            flags="dmsquash"
+            flags="overlay"
             hybridpersistent_filesystem="ext4"
             hybridpersistent="true"
-            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=qemu"
-            bootkernel="custom"
-            devicepersistency="by-uuid"
-        >
-            <systemdisk>
-                <volume name="home"/>
-                <volume name="root"/>
-                <volume name="tmp"/>
-                <volume name="opt"/>
-                <volume name="srv"/>
-                <volume name="boot/grub2/i386-pc"/>
-                <volume name="boot/grub2/x86_64-efi" mountpoint="boot/grub2/x86_64-efi"/>
-                <volume name="boot/writable"/>
-                <volume name="usr/local"/>
-                <volume name="var" copy_on_write="false"/>
-            </systemdisk>
-            <machine memory="512" guestOS="suse" HWversion="4">
-                <vmdisk id="0" controller="ide"/>
-                <vmnic driver="e1000" interface="0" mode="bridged"/>
-            </machine>
-        </type>
+            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0"
+            devicepersistency="by-uuid"/>
     </preferences>
+    <users>
+         <user password="suse1234" home="/root" name="root" groups="root" pwdformat="plain"/>
+	 <user password="vagrant" home="/home/vagrant" name="vagrant" groups="vagrant" pwdformat="plain"/>
```

Comment:
is there a reason why we're adding the vagrant user here instead of doing that on the `config.sh` script along with other vagrant-related operations?

Reply:
Or in the Vagrant preferences.

Reply:
> is there a reason why we're adding the vagrant user here instead of doing that on the `config.sh` script along with other vagrant-related operations?

I will remove that for non-vagrant image and put that into build script.

Reply:
fixed in the last commit and vagrant user only for vagrant image